lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -110,47 +125,52 @@ class _FindClosestFacilityFromPointSampleState
       geometry2: incidentsLayer.fullExtent!,
     );
 
+    // Set the viewpoint geometry on the map view controller
     _mapViewController.setViewpointGeometry(mapExtent, paddingInDiPs: 30);
 
+    // Generate the closest facility parameters
     _closestFacilityParameters = await generateClosestFacilityParameters(
         facilitiesLayer, incidentsLayer);
 
-    setState(() => _isInitialized = true);
+    // Set the initialized flag to true
+    setState(() => _initialized = true);
   }
 
   FeatureLayer buildFeatureLayer(Uri tableUri, Uri imageUri) {
+    // Create a feature table and feature layer for the facilities or incidents.
     final featureTable = ServiceFeatureTable.withUri(tableUri);
-    final featureLayer = FeatureLayer.withFeatureTable(featureTable);
-    final markerSymbol = PictureMarkerSymbol.withUrl(imageUri);
-    markerSymbol.width = 30;
-    markerSymbol.height = 30;
-    featureLayer.renderer = SimpleRenderer(symbol: markerSymbol);
+    final markerSymbol = PictureMarkerSymbol.withUrl(imageUri)
+      ..width = 30
+      ..height = 30;
+    final featureLayer = FeatureLayer.withFeatureTable(featureTable)
+      ..renderer = SimpleRenderer(symbol: markerSymbol);
 
     return featureLayer;
   }
 
   Future<ClosestFacilityParameters> generateClosestFacilityParameters(
       FeatureLayer facilitiesLayer, FeatureLayer incidentsLayer) async {
+    // Create query parameters to get all features.
     final featureQueryParams = QueryParameters()..whereClause = '1=1';
-
-    final parameters = await _closestFacilityTask.createDefaultParameters();
-    parameters.setFacilitiesWithFeatureTable(
-      featureTable: facilitiesLayer.featureTable! as ArcGISFeatureTable,
-      queryParameters: featureQueryParams,
-    );
-    parameters.setIncidentsWithFeatureTable(
-      featureTable: incidentsLayer.featureTable! as ArcGISFeatureTable,
-      queryParameters: featureQueryParams,
-    );
+    // Create default parameters for the closest facility task.
+    final parameters = await _closestFacilityTask.createDefaultParameters()
+      ..setFacilitiesWithFeatureTable(
+        featureTable: facilitiesLayer.featureTable! as ArcGISFeatureTable,
+        queryParameters: featureQueryParams,
+      )
+      ..setIncidentsWithFeatureTable(
+        featureTable: incidentsLayer.featureTable! as ArcGISFeatureTable,
+        queryParameters: featureQueryParams,
+      );
 
     return parameters;
   }
 
   Future<void> solveRoutes() async {
```

Comment:
```suggestion
  void solveRoutes() async {
```
We're not calling `await` anywhere so don't need the Future part